sensor/common/compliance/nodescan_handler_impl.go

```diff
@@ -44,9 +43,6 @@ func (c *nodeScanHandlerImpl) ProcessMessage(_ *central.MsgToSensor) error {
 }
 
 func (c *nodeScanHandlerImpl) run() {
-	defer c.stoppedC.SignalWithError(c.stopC.Err())
-	defer close(c.toCentral)
```

Comment:
do we not want to close this anymore?

Reply:
We do want to close this, but I think this is kind of hard here. For this to happen, we would need to make the component restartable. Let me give it one more try, maybe I will get it this time :)

Reply:
Addressed in https://github.com/stackrox/stackrox/commit/3f47d78f5160018d11d026c4629931653863bcb0 and updated the PR description. PTAL